Dockerfile

```diff
@@ -6,4 +6,4 @@ WORKDIR /data
 
 RUN pip install -e .
 
-CMD ["howfairis"]
+CMD howfairis "https://github.com/${INPUT_REPOSITORY}"
```

Comment:
Nice work Carlos! For best practices in writing Dockerfiles, cmd is better represented by

```
CMD [ "howfairis", "https://github.com/${INPUT_REPOSITORY}"]
```